locales/en-US/server.ftl

```diff
@@ -15,7 +15,7 @@ buttonSignIn =
     .title = Sign In
 screenshotsLogo =
     .title = Screenshots Home
-bannerMessage = Sign in or sign up to access your shots across devices and save your favorites forever.
+bannerMessage = <a>Sign in or sign up</a> to access your shots across devices and save your favorites forever.
```

Comment:
Sorry, this [kind of changes](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings) require a new string ID